package.json

```diff
@@ -90,7 +91,7 @@
     "build-size": "rollup -c --silent && gzip-size micromark.min.js && gzip-size --raw micromark.min.js",
     "build": "npm run build-workspace && npm run build-monorepo && npm run build-size",
     "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix",
-    "pretest-fuzz": "patch-package --patch-dir script/patches && node script/generate-fixtures.js",
+    "pretest-fuzz": "patch-package --patch-dir script/patches && echo \"import {micromark} from 'micromark'; export {micromark}\" | esbuild --format=cjs --platform=node --bundle > test/fuzz-bundle.cjs && node script/generate-fixtures.js",
```

Comment:
`import {micromark} from 'micromark'; export {micromark}` seems unneeded? can’t the source be bundled? 🤔

Reply:
I use this as a short hand to find the right file which should be bundled.
Following node's folder rules.

It could be pointed at a specific file, but then we'd need to hard code a specific (potentially changing) path.

Reply:
Doesn't esbuild look into node_modules, and nom workspaces put mm there tho?

Reply:
right, but it needs an entrypoint, and I have not found an option to bundle just a specific module yet, so I create a sample file to start that process.

Reply:
Oh I though esbuild did allow "foo" if it was in node_modules! OK!

Reply:
Maybe?
Trying to do a clean build and test this now and discovered npm is down. https://status.npmjs.org/incidents/4p6mgfgs0nxt
May have to wait to test

Reply:
Expand the details here: https://mdxjs.com/docs/troubleshooting-mdx/#esm.
That’s picking up `node_modules/@mdx-js/mdx`, hence when I was guessing that an extra file wouldn’t be needed?

Reply:
updated in 1336066

Reply:
ah nice, so it does work, right? Perfect! :+1: